primo/opt_model/model_options.py

```diff
@@ -322,3 +322,43 @@ def solve_model(self, **kwargs):
 
         # In all other cases, return the optimal campaign
         return self._opt_model.get_optimal_campaign()
+
+    def update_cluster(self, add_widget_return):
+        """
+        Updates the campaign candidates by changing the cluster numbers for specific wells.
+
+        Parameters
+        ---------
+        add_widget_return : AddWidgetReturn
+            An AdWidgetReturn object which includes information on wells selected to add
+            to the existing optimal P&A projects
+        """
+        existing_clusters = add_widget_return.existing_cluster
+        new_clusters = add_widget_return.new_cluster
+        wd = self.config.well_data
+        col_names = wd.col_names
+
+        if existing_clusters == new_clusters:
```

Comment:
Really minor but can be simplified by skipping lines 341-342 and replacing 343 as:
`if existing_clusters != new_clusters`